prac_07/project_management.py

```diff
@@ -0,0 +1,114 @@
+"""
+Project management - Prac 07
+Estimate time = 1hr 20mins
+Actual time = 2hrs 35mins
+"""
+
+from prac_07.project import Project
+import datetime
+
+
+MENU = "- (L)oad Projects\n- (S)ave Projects\n- (D)isplay Projects\n- (F)ilter projects by date\n- (A)dd new project\n- (U)pdate Project\n- (Q)uit Project"
+FILENAME = "projects.txt"
+
+
+def main():
+
+    projects = load_projects(FILENAME)
+    print(MENU)
+    menu_choice = input(">>> ").upper()
+    while menu_choice != "Q":
+        if menu_choice == "L":
+            filename = input("File to load from: ")
+            load_projects(filename)
+        elif menu_choice == "S":
+            save_projects(projects, FILENAME)
+        elif menu_choice == "D":
+            display_projects(projects)
+        elif menu_choice == "F":
+            filter_projects(projects)
+        elif menu_choice == "A":
+            add_new_project(projects)
+        elif menu_choice == "U":
+            update_project(projects)
+        else:
+            print("Invalid menu choice")
+        print(MENU)
+        menu_choice = input(">>> ").upper()
+    save_projects(projects, FILENAME)
+
+
+def load_projects(filename):
+    projects = []
+    in_file = open(filename, "r", encoding="utf-8-sig")
+    in_file.readline()
+    for line in in_file:
+        parts = line.strip().split("	")
+        parts[2] = int(parts[2])
+        parts[3] = float(parts[3])
+        parts[4] = int(parts[4])
```

Comment:
these can be done in the below "project =" line: 
project = Project(..., ..., int(parts[2]), float(parts[3], int(parts[4]))